devel.config

```diff
@@ -1,16 +1,21 @@
-backend_port = 7001
-frontend_port = 7002
+command: "run"
```

Comment:
Ok, cool, that's good enough for now. I almost always run `run` anyway. It's not like Thentos is git or cabal.